content/doc/book/getting-started/installing.adoc

```diff
@@ -0,0 +1,177 @@
+---
+layout: section
+---
+:doctitle: Installing Jenkins
+:notitle:
+:description:
+:author: Liam Newman
+:email: jenkinsci-users@googlegroups.com
+:sectanchors:
+:toc: left
+
+[NOTE]
+====
+This is still very much a work in progress
+====
+
+[IMPORTANT]
+====
+This section is part of _Getting Started_.
+It provides a golden path for a *basic* jenkins configuration on a number of platforms.
```

Comment:
`jenkins` should be capitalized always. Also "golden path" is kind of a weird metaphor, perhaps just drop it.